Headers/AppKit/AppKit.h

```diff
@@ -234,6 +234,7 @@
 #import <AppKit/NSTableHeaderView.h>
 #import <AppKit/NSTableView.h>
 #import <AppKit/NSTabView.h>
+#import <AppKit/NSTabViewController.h>
```

Comment:
Shouldn't we include the NSSplitViewController here as well?

Reply:
Fixed